src/ATen/function_wrapper.py

```diff
@@ -521,6 +537,19 @@ def emit_body(env, option):
             if arg['type'] == 'THSize*':
                 scalar_check_is_from_size = True
                 scalar_check = '{}.size() == 0'.format(arg['name'])
+
+            wrap_dim_arg = arg.get('wrap_dim', None)
+            if wrap_dim_arg is not None:
+                # wrap_dim specification can have (add) expressions, e.g. self+1
```

Comment:
Do we have a test case for an example with this expression type?

Reply:
yes, the unsqueeze ones.